workers_test.go

```diff
@@ -154,6 +197,41 @@ func TestWorkers(t *testing.T) {
 	}
 }
 
+func TestWorkersFinish(t *testing.T) {
+	ctx := context.Background()
+
+	var wg = new(sync.WaitGroup)
+	workerOne := NewRunner(ctx, NewWorkerOne(), 1000).Start()
+	workerTwo := NewRunner(ctx, NewWorkerTwo(), 1000).InFrom(workerOne).Start()
+
+	wg.Add(1)
+	go func() {
```

Comment:
The go scheduler will screw up this test: there are no guarantees that we will send any values before we call `workerOne.Wait()`, even though this goroutine is called before we call wait